### ISSUE TITLE
There is no clear difference between read and unread mail.

### ISSUE THREAD

**Issue by mytvac** [mozilla/kitsune#6424]
There is no clear difference between read and unread mail.

**Comment by escattone** [comment]
@mytvac Is your question related to a Mozilla product? Perhaps your question is related to Thunderbird? This repository is only for the code that powers the Mozilla support website. For help or to ask a question related to Thunderbird, you can start [here](https://support.mozilla.org/en-US/products/thunderbird).

**Comment by wsmwk** [comment]
@escattone I agree this is more likely to be a Thunderbird issue - mobile or desktop
